packages/wonder-blocks-core/components/text.md

```diff
@@ -0,0 +1,40 @@
+```js
+const {StyleSheet} = require("aphrodite");
+
+const styles = StyleSheet.create({
+    container: {
+        padding: 32,
+        backgroundColor: "plum",
+    },
+    text: {
+        fontFamily: "sans-serif",
+        fontSize: 24,
+    },
+});
+
+<View style={styles.container}>
+    <Text style={styles.text}>
+        Hello, world!
+    </Text>
+    <View style={[
+        styles.container,
+        {backgroundColor: "goldenrod", padding: 4},
+    ]}>
+        Even with an array and inline styles!
+    </View>
```

Comment:
We'd probably want a more `Text` specific example here, especially one that better motivates when you'd want to use `Text` vs. our other typography components, e.g. showing how to make a word within a sentence bold or a different color.

Reply:
Hmm, ok - I think I'll hold off on that until we improve the Text docs in general.